io.edgehog.devicemanager.WiFiAP.json

```diff
@@ -0,0 +1,42 @@
+{
+  "interface_name": "io.edgehog.devicemanager.WiFiAP",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "aggregation": "object",
+  "mappings": [
+    {
+      "endpoint": "/ap/channel",
+      "type": "integer",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "The channel over which the client is communicating with the access point."
+    },
+    {
+      "endpoint": "/ap/essid",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Extended Service Set Identification of the current AP"
+    },
+    {
+      "endpoint": "/ap/macAddress",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Mac address of the current AP"
+    },
+    {
+      "endpoint": "/ap/rssi",
+      "type": "integer",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Received Signal Strength Indicator of the current AP"
```

Comment:
which unit? is it dBm RSSI? I think we should document it, and maybe provide a few examples. So let's add some documentation here too.